### DISCUSSION TITLE
Wofür ist der Widget Label, Unterschied zu Titel?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1173] [Q&A]
![image](https://user-images.githubusercontent.com/84818934/141647961-eb2391d6-4843-465a-bbaf-60247245a241.png)
Was ist der Hintergrund? Wann braucht man den Label und wann den Titel?

**Comment by Zefau** [comment]
Siehe https://github.com/Zefau/ioBroker.jarvis/issues/1033

Nur für eine Unterscheidung in der Konfiguration.
